ci/lib/install_functions.inc.sh

```diff
@@ -535,30 +550,43 @@ linux_install() {
 msys_install() {
   local packages=(
     tar
-    zlib-devel
-    libbz2-devel
     git
     automake
     autoconf
     libtool
     automake-wrapper
     gnupg2
     make
-    pkg-config
+    pkg-config 
     mingw64/mingw-w64-x86_64-cmake
-    mingw64/mingw-w64-x86_64-gcc
-    mingw64/mingw-w64-x86_64-json-c
     mingw64/mingw-w64-x86_64-python3
   )
 
+  if [ "${CC-gcc}" = "gcc" ]; then
```

Comment:
ditto

Reply:
CC="" will force cmake default which would be gcc. If CC is unset it will also force cmake default but it may fail at this line depending on shell settings.